backends/tfhe-cuda-backend/cuda/include/bootstrap.h

```diff
@@ -5,6 +5,7 @@
 #include <cstdint>
 
 enum PBS_TYPE { MULTI_BIT = 0, LOW_LAT = 1, AMORTIZED = 2 };
```

Comment:
When renaming, we'll end up with:
`PBS_MULTI_BIT`, `PBS` and `PBS_AMORTIZED` maybe? What's the difference between PBS type and PBS variant by the way? I would have expected type to be CLASSICAL or MULTI_BIT and variant to be DEFAULT, CG and TBC?

Reply:
Since we have no short term plans to use the amortized PBS we could have `MULTI_BIT` and `CLASSICAL` indeed. And you are right about the variant. It refers to default, fast (or cg), or TBC.